src/tokens.js

```diff
@@ -6,14 +6,14 @@
  * @flow
  */
 
-import type {Reducer, StoreEnhancer, Store} from 'redux';
+import type {Reducer, StoreEnhancer} from 'redux';
 
 import {createToken} from 'fusion-core';
 import type {Token, Context} from 'fusion-core';
 
 import type {ReactReduxServiceType} from './types.js';
 
-type InitialStateType<S, A, D> = (ctx?: Context) => Promise<Store<S, A, D>>;
+type InitialStateType = (ctx?: Context) => Promise<*> | *;
```

Comment:
Hmm, I'm not sure we should be using `*` for any new types.  It looks like it is being [deprecated](https://github.com/facebook/flow/releases/tag/v0.72.0).  If we aren't sure what type to use here (i.e. don't use redux `Store`) we should instead of generics:

```
type InitialStateType<TState> = (ctx?: Context) => Promise<TState> | TState
```

Reply:
Yeah, that's better - thanks.